src/coreclr/jit/lsrabuild.cpp

```diff
@@ -1899,6 +1899,18 @@ void LinearScan::buildPhysRegRecords()
         RegRecord* curr = &physRegs[reg];
         curr->regOrder  = (unsigned char)i;
     }
+#if defined(TARGET_AMD64)
+    if (compiler->DoJitStressEvexEncoding())
+    {
+        for (unsigned int i = 0; i < lsraRegOrderFltSize; i++)
```

Comment:
Should this be `lsraRegOrderUpperFltSize` os something similar?

Reply:
hoist the upper register array declaration, and define the corresponding length as `lsraRegOrderUpperFltSize`.